src/service/validations/signup.js

```diff
@@ -0,0 +1,185 @@
+
```

Comment:
those validations are a too much of vanilla javascript maybe take a look in this blog https://dev.to/codebucks/form-validation-in-reactjs-by-building-reusable-custom-hook-1bg7, or consider a third party form validation library